src/components/TicketModal/index.jsx

```diff
@@ -78,7 +90,17 @@ export const TicketModal = (props) => {
             <Card.Content>
               <div className="ticket-card-bottom-header">
                 NOTES {notes ? notes.length : 0}
+                <div onClick={toggleShowAddNote}>
+                  <i className="fas fa-plus-circle icon-inline-space" />
+                </div>
               </div>
+              {showAddNote ?
+                <AddNote
+                  handleAddNote={props.handleAddNote}
+                  toggleShowAddNote={toggleShowAddNote}
+                  ticketID={id}
+                />
+                : null}
               {notes ? (
                 notes.map((note) => {
```

Comment:
Do we want to order these with the most recent on the top? It might make more sense since the input field is above the list

Reply:
We can order the notes with the most recent on the top, but I feel like that would probably be a backend thing.  I assume we can change the order in which they are take from the database, rather than reorder them in the frontend, if that makes sense.  I can create a ticket for that if you'd like.  Thanks!

Reply:
Yeah that would be great! Thanks